gltf-importer/src/lib.rs

```diff
@@ -241,73 +152,58 @@ fn load_external_buffers(
         .collect()
 }
 
-fn validate_standard(json: &gltf::json::Root, config: &Config) -> Result<(), Error> {
+fn validate_standard(
```

Comment:
It's an implementation detail that I'd rather keep private for now. Is there any reason why it should be exported?